src/base/static/js/views/app-view.js

```diff
@@ -461,7 +459,7 @@ module.exports = Backbone.View.extend({
   },
   onToggleSidebarVisibility: function() {
     $("body").toggleClass("right-sidebar-visible");
-    this.mapView.map.invalidateSize();
+    store.dispatch(setMapSizeValidity(false));
```

Comment:
Action to signal the need for map re-centering. (To be replaced later by open/closed state stored in a UI duck.)

Reply:
This is janky but good enough for now.